frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-inventory-card.tsx

```diff
@@ -116,7 +116,7 @@ export const VMInventoryCard: React.FC<VMInventoryCardProps> = () => {
         {diskCount > 0 && (
           <InventoryItem
             isLoading={isLoading}
-            title={DiskType.DISK.toString()}
+            title={t('kubevirt-plugin~Disk')}
```

Comment:
Note: not using plural translation here because the `InventoryItem` does none i18n pluralization and adds `s`, it we pluralize using `i18n` we will get an extra s `<translated plural>s` that comes from `InventoryItem`

Reply:
you can pass `titlePlural`

Reply:
added titlePlural